### DISCUSSION TITLE
Can't run katana on github action.

### DISCUSSION THREAD

**Discussion by syhbt** [projectdiscovery/katana#642] [Q&A]
Hi guys, I tried to create a github action repository for Katana and all Katana installations and binaries run normally, but Katana doesn't display results, like it's stuck and can't make requests to the target. Does GitHub have restrictions on using crawler tools on actions? or is this just a problem with the katana?

```
Run username/katana-action@main
Download and install Katana 1.0.4
/home/runner/work/workspace/workspace/katana -u=http://testphp.vulnweb.com/ -o=katana.log

   __        __                
  / /_____ _/ /____ ____  ___ _
 /  '_/ _  / __/ _  / _ \/ _  /
/_/\_\\_,_/\__/\_,_/_//_/\_,_/							 

		projectdiscovery.io

[INF] Current katana version v1.0.4 (latest)

```

I ended up 1 hour with no results. 😭

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
